try-it.md

```diff
@@ -131,7 +130,7 @@ definition file used to create these host objects is in `bmhosts_crs.yaml`.
 $ kubectl get baremetalhosts -n metal3 -o wide
 NAME     STATUS   STATE   CONSUMER   BMC                         HARDWARE_PROFILE   ONLINE   ERROR
 node-0   OK       ready              ipmi://192.168.111.1:6230   unknown            true
-node-1   OK       ready              ipmi://192.168.111.1:6231   unknown            true
```

Comment:
I think this part was showing unprovisioned (i.e. `Ready` BareMetalHosts), maybe that should be kept as it is?

Reply:
I have accidentally copied the the wrong output, actually before provisioning it looks like this:
```sh
$ kubectl get baremetalhosts -n metal3 -o wide
NAME     STATUS   STATE   CONSUMER   BMC                         HARDWARE_PROFILE   ONLINE   ERROR
node-0   OK       ready              ipmi://192.168.111.1:6230   unknown            true
node-1   OK       ready              redfish+http://192.168.111.1:8000/redfish/v1/Systems/a1cd44ba-c6db-49ac-bb07-56d4fbc5380f   unknown            true
```